.prettierignore

```diff
@@ -1,17 +1,17 @@
 # The GH actions don't seem to compile and verify themselves well when Prettier is applied to them
```

Comment:
These should be uncommented

Reply:
The `react-native-x-maps` library is working without these ignores. Removing the file `.prettierignore` all together.